src/main/java/org/openmrs/maven/plugins/Delete.java

```diff
@@ -25,13 +25,8 @@
      */
     private String serverId;
 
-    /**
-     * @required
-     * @component
-     */
-    Wizard wizard;
-
     public void execute() throws MojoExecutionException, MojoFailureException {
+        initTask();
```

Comment:
I felt that invoking `initTask()` in every task is poor design, but didn't come up with better idea yet, thanks for tip!